compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/WindowComposeSceneLayer.desktop.kt

```diff
@@ -78,49 +83,53 @@ internal class WindowComposeSceneLayer(
         dialog.contentPane = it
     }
 
+    private val windowPositionListener = object : ComponentAdapter() {
+        override fun componentMoved(e: ComponentEvent?) {
+            onChangeWindowPosition()
+        }
+    }
+
+    private val dialogFocusListener = object : WindowFocusListener {
+        override fun windowGainedFocus(e: WindowEvent?) = Unit
+        override fun windowLostFocus(e: WindowEvent?) {
+            // Use this as trigger of outside click
+            outsidePointerCallback?.invoke(PointerEventType.Press)
+            outsidePointerCallback?.invoke(PointerEventType.Release)
+        }
+    }
+
     private var _mediator: ComposeSceneMediator? = null
+    private var outsidePointerCallback: ((eventType: PointerEventType) -> Unit)? = null
 
     override var density: Density = density
         set(value) {
             field = value
-            // TODO: Pass it to mediator/scene
+            _mediator?.onChangeDensity(value)
         }
 
     override var layoutDirection: LayoutDirection = layoutDirection
         set(value) {
             field = value
-            // TODO: Pass it to mediator/scene
+            _mediator?.onChangeLayoutDirection(value)
         }
 
-    override var focusable: Boolean = focusable
-        set(value) {
-            field = value
-            // TODO: Pass it to mediator/scene
+    override var focusable
+        get() = true
+        set(_) {
+            // Ignore the value - window is always focusable
```

Comment:
As far as I remember `focusable = false` means that it doesn't receive events. And that is usually possible to achieve with window-popups. Maybe we can achieve this in general windows as well 🤔

Reply:
Let's revert to TODO for now